.github/workflows/scripts/docs/build-docs.sh

```diff
@@ -10,7 +10,7 @@ docker run \
   --user "$(id -u):$(id -g)" \
   --volume "${PWD}:/docs" \
   --name "build-docs" \
-  squidfunk/mkdocs-material:8.2.8 build --strict
+  squidfunk/mkdocs-material:8.3.4 build --strict
```

Comment:
```suggestion
  squidfunk/mkdocs-material:8.3.5 build --strict
```

8.3.5 Just got released